Homework_11/CircusManagement/src/CircusPerformer.java

```diff
@@ -0,0 +1,59 @@
+public class CircusPerformer {
+    private String name;
+    private String act;
+    private int experience;
+
+
+    public CircusPerformer(String name, String act, int experience) {
+        this.name = name;
+        this.act = act;
+        this.experience = experience;
+    }
+
+    public String getName() {
+        System.out.println("Name :" + name);
```

Comment:
usually we don't print anything to console within getter methods - their purpose is simple -> just return the value of a corresponding private field

Sometimes for debugging purpose println() can be added but then removed before submitting a PR